backend/e4h-services/ingestion-service/app/utils/mdms_client.py

```
@@ -30,9 +59,15 @@ def fetch_schema(self, request_info: RequestInfo, schema_code: str) -> 'Ingestio
         headers = {
             "Accept": "application/json, text/plain, */*",
         }
-        response = requests.post(url, headers=headers, json=payload)
-        # return convert_json_to_object(response.text)
-        return IngestionSchemaResponse.model_validate(response.json())
+        try:
+            response = requests.post(url, headers=headers, json=payload)
```

Ruff (0.16.1)

[error] Probable use of `requests` call without timeout

(S113)